### ISSUE TITLE
`pixiv.net` -> `phixiv.net` replacement no longer working

### ISSUE THREAD

**Issue by ryln7218** [podaboutlist/linkfix-for-discord#23]
As the title suggests, replying to pixiv.net links with either images of the arts/files in the link or even a simple phixiv.net link would be great! The former might be hard to implement (the only bot I saw that could do this was Boe Tea) but hopefully the latter follows the same logic as all the other links!

Thanks

**Comment by ryln7218** [comment]
Also sorry I don't know how to add tags to this post

**Comment by RalphORama** [comment]
I have not used Pixiv before but I'll look into it! Is there a separate fx/vxtwitter site or are you talking about direct image links?

**Comment by RalphORama** [comment]
After a cursory look it doesn't appear there are any sites like `vxpixiv.net` or `fxpixiv.net` that do this already. I could *probably* add a feature that loads the page and extracts the direct image URL, but #19 is currently taking up most of my time. Once database support is finished I'll look into this again!

**Comment by ryln7218** [comment]
I was talking about direct image links but if that is too hard, similar to
fx/vx twitter phixiv exists for pixiv (note the addition of 'h') although
it's not a large improvement from just pixiv embeds...
(1st image just pixiv, 2nd image phixiv)

**Comment by RalphORama** [comment]
Unfortunately it looks like they have image hotlinking protection in place so direct image links won't load on Discord. I will add functionality for phixiv, but it would be fairly complicated to grab full resolution images. 

The other bot you mentioned probably downloads/re-uploads the image using a burner account which is way more complicated than what I'd like to implement lol

**Comment by RalphORama** [comment]
Functionality added! I don't use Pixiv very much so if you encounter any issues with LinkFix and Pixiv, please feel free to open another issue

**Comment by ryln7218** [comment]
It worked for a bit when you added it but now it doesn't seem to work...?
sent a link such as "https://www.pixiv.net/artworks/114853941" on discord where the bot has access to and no reply with a phixiv link....
All other features (twitter --> fx) still work as intended.

**Comment by RalphORama** [comment]
Think I might have screwed up the regex at some point. I'll look into it. Thanks for letting me know!

**Comment by RalphORama** [comment]
@ryln7218 just pushed an update that should fix Pixiv replacements. thanks for letting me know about the issue!
